meta-freescale-3rdparty-mbl/conf/machine/imx7d-pico-mbl.conf

```diff
@@ -45,7 +45,7 @@ KERNEL_IMAGETYPE = "fitImage"
 SPL_BINARY=""
 UBOOT_EXTLINUX = "0"
 UBOOT_CONFIG = ""
-UBOOT_MACHINE = "pico-pi-imx7d_defconfig"
```

Comment:
The commit comment doesn't include the github PR explanation for why these changes are needed:

```
use upstream u-boot for pico.

DCD file cannot be merge to mainline u-boot. Because there are two version pico board with 1GB/512MB DRAM. So u-boot guys insist u-boot SPL should be used to initialize DRAM, instead of DCD data. We have to carry it in recipe.
```

Please can you update the commit comment to include more details? Thanks.

Reply:
Already updated.

Reply:
thanks